art_internal_impl.hpp

```diff
@@ -110,7 +129,8 @@ class [[nodiscard]] basic_leaf final : public Header {
 
 #endif  // UNODB_DETAIL_WITH_STATS
 
-  [[gnu::cold]] UNODB_DETAIL_NOINLINE void dump(std::ostream &os) const {
+  [[gnu::cold]] UNODB_DETAIL_NOINLINE void dump(std::ostream &os,
+                                                bool /*recursive*/) const {
```

Comment:
Do not name unused parameters:
```suggestion
                                                bool) const {
```